notebooks/akride_explore_dataset.ipynb

```diff
@@ -262,8 +262,7 @@
     "job_spec = client.create_job_spec(dataset=ds, job_type=JobType.EXPLORE ,\n",
```

Comment:
@optimus-kart Is there a SDK change corresponding to this change to remove catalog_name?

Reply:
there is a catalog_table in create_job_spec which is an object and not string(catalog_table=CatalogTable(table_name=view_cl, is_view=True)), catalog_name is incorrect